Description: Trivial spelling error
 Code comment has a mis-spelled word.
Author: Matthew Grant
Last-Update: 2011-10-03

Index: ipsec-tools/src/libipsec/ipsec_strerror.h
===================================================================
--- ipsec-tools.orig/src/libipsec/ipsec_strerror.h	2011-10-03 19:43:47.000000000 +1300
+++ ipsec-tools/src/libipsec/ipsec_strerror.h	2011-10-03 20:58:43.672162611 +1300
@@ -54,7 +54,7 @@
 #define EIPSEC_INVAL_KEYLEN	14	/*invalid key length*/
 #define EIPSEC_INVAL_FAMILY	15	/*invalid address family*/
 #define EIPSEC_INVAL_PREFIXLEN	16	/*SPI range violation*/
-#define EIPSEC_INVAL_DIR	17	/*Invalid direciton*/
+#define EIPSEC_INVAL_DIR	17	/*Invalid direction*/
 #define EIPSEC_INVAL_SPI	18	/*invalid prefixlen*/
 #define EIPSEC_NO_PROTO		19	/*no protocol specified*/
 #define EIPSEC_NO_ALGS		20	/*No algorithm specified*/
